jbpm-flow/src/main/java/org/jbpm/process/instance/impl/FeelReturnValueEvaluatorListener.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.process.instance.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.kie.dmn.api.feel.runtime.events.FEELEvent;
+import org.kie.dmn.api.feel.runtime.events.FEELEventListener;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class FeelReturnValueEvaluatorListener implements FEELEventListener {
```

Comment:
Considering that it collects only `ERROR` events, what about rename it as
```suggestion
public class FeelErrorEvaluatorListener implements FEELEventListener {
```